src/app/containers/SocialEmbed/index.stories.jsx

```diff
@@ -1,12 +1,86 @@
-import { storiesOf } from '@storybook/react';
+import React from 'react';
```

Comment:
Big rewrite on this one as it doesn't seem to work at all - broken on latest: https://bbc.github.io/simorgh/?path=/story/containers-cps-social-embed-canonical--default